### ISSUE TITLE
#2 Create an Inviting Css Layout

### ISSUE THREAD

**Issue by Kurtiscwright** [suwebdev/small-dawn#16]
[Content User Story #4](https://github.com/suwebdev/small-dawn/wiki/Content-User-Story)

**User Story Size:** Md

**Reason for adding to the Sprint:**
If we do not have a common theme/layout then the site won't stand out vs others

**The focus of this task:**
Is to create and put into place a CSS design that will be ubiquitous to the site.

**How this fits into the build process:**
It helps flush out the look of the site and will help the developers visualize the end goal.

**In this sprint because:**
It is needed for all sites in the modern web experience.

**Comment by Victa727** [comment]
I agree that it's important to have a theme decided.  Maybe even a prebuilt bootstrap template.
